src/backend/dx11/src/conv.rs

```diff
@@ -225,6 +111,321 @@ pub fn map_format(format: Format) -> Option<DXGI_FORMAT> {
     Some(format)
 }
 
+#[derive(Debug, Clone)]
+pub struct DecomposedDxgiFormat {
+    pub typeless_format: DXGI_FORMAT,
```

Comment:
whoa, this is looking very thorough!
Perhaps, we can omit `_format` suffices in the fields?